src/OrleansCodeGenerator/CodeGeneratorCommon.cs

```diff
@@ -72,6 +72,18 @@ public static GeneratedAssembly CompileAssembly(GeneratedSyntax generatedSyntax,
             // Generate the code.
             var options = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
 
+#if NETSTANDARD
```

Comment:
hmmm, do you think so? it's not something that we need to address ASAP, is it? We are still depending on this external issue.